src/es/components/controllers/events/Events.js

```diff
@@ -211,6 +212,13 @@ export default class Events extends Shadow() {
         }))
       })
     }
+
+     // set Monday as first day in Datepickr
```

Comment:
use the "this.getDateOptionEventsListener'-Object to return this in the object. A controller should never use a querySelector.

Reply:
i fixed it here... so you should be able to delete L215-222

Reply:
https://github.com/mits-gossau/web-components-toolbox/commit/bdebe1974569d1500b0117af08cb0b8061640ec2#diff-28617f8fdaae0e1e22dbbf887a04b79c8920e9146528481d947504e7e538356fR201

Reply:
Removed